<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BibTeXDialog</class>
 <widget class="QDialog" name="BibTeXDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">&lt;generated in source&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,2">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Insert in File:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Entry Type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QListWidget" name="fileList"/>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QListWidget" name="typeList"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Fields:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTableWidget" name="fieldTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Field</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="optFieldCheckBox">
     <property name="text">
      <string>Insert Empty, Optional Fields</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BibTeXDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BibTeXDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
